lib/prefix.h

```diff
@@ -383,6 +383,8 @@ static inline void ipv4_addr_copy(struct in_addr *dst,
 #define IPV4_NET127(a) ((((uint32_t)(a)) & 0xff000000) == 0x7f000000)
 #define IPV4_LINKLOCAL(a) ((((uint32_t)(a)) & 0xffff0000) == 0xa9fe0000)
 #define IPV4_CLASS_DE(a) ((((uint32_t)(a)) & 0xe0000000) == 0xe0000000)
+#define IPV4_CLASS_D(a) ((((uint32_t)(a)) & 0xf0000000) == 0xe0000000)
+#define IPV4_CLASS_E(a) ((((uint32_t)(a)) & 0xf0000000) == 0xf0000000)
```

Comment:
I'm going to make the same comment here that I made on Dave Täht's pull request quite some while ago:

please split this up into 2 separate changes.

1. create a new macro/helper function, something like `ipv4_valid_unicast` — *without* changing to allow class E
2. allow class E in this macro with a change in only one place

The reason here is that 1. can be merged immediately because it is a good improvement in code quality and structure, and 2. will need discussion and maybe a config switch.

Reply:
@eqvinox a global switch like in `lib/command.c`?